### PR TITLE
Make Base64URL encoding modular

### PR DESCRIPTION
**Pull request by przemyslawzalewski**
This is a BREAKING CHANGE - base64url imports must be updated and there was also some behavioral change (e.g. Base64 always includes padding characters and Base64URL never does).

### CODE REVIEW

File: encoding/base64url/toBase64.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "toBase64",
+  "description": "TODO: Fill short description here.",
```
Comment (marekrozmus) [inline encoding/base64url/toBase64.json]: no description provided

File: encoding/byteString/index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import _from from "./from.js";
```
Comment (marekrozmus) [inline encoding/byteString/index.js]: why underscore?

Comment (przemyslawzalewski) [reply encoding/byteString/index.js]: `I was pretty sure `from` was a reserved keyword just like `function`.  I am gonna investigate that.

Comment (przemyslawzalewski) [reply encoding/byteString/index.js]: Yes, you were right. `from` is not a reserved keyword and does not need aliasing.

File: encoding/base64url/toBase64.json (hunk 1 above)
Comment (przemyslawzalewski) [reply encoding/base64url/toBase64.json]: I've fixed that.

**Pull request merged**